assets/admin/students/student-modal/course-list.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { CheckboxControl, Spinner } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Course list.
+ */
+export const CourseList = () => {
+	const [ isFetching, setIsFetching ] = useState( true );
+	const [ courses, setCourses ] = useState( [] );
+
+	// Fetch the courses.
+	useEffect( () => {
+		setIsFetching( true );
+
+		apiFetch( {
+			path: '/wp/v2/courses?per_page=100',
```

Comment:
Are we going to list only the first 100 items?

Reply:
Oh yes, sorry. I meant to put this in the description. WordPress will only return a maximum of 100 courses. We had this same problem when trying to fetch existing questions in the lesson editor. This means users will need to search to find any other courses. I don't think this will be that big a deal though, as I don't expect too many sites have 100+ courses.

Reply:
@donnapep Should we add some text explaining it just to avoid possible misunderstandings?